### PR TITLE
added a test to verify 50% after cliff and 50% at next one step

### PR DESCRIPTION
**Pull request by ismaelmk**

### CODE REVIEW

File: test/TestStepVestingPromo.js [hunk 1]
```diff
@@ -37,25 +37,74 @@ contract('StepVesting', function ([_, owner, beneficiary]) {
...
+    va.should.bignumber.equal(new BigNumber(0));
+
+    let now = this.start + this.cliffDuration + 2;
+
+    await timeUtils.increaseTimeTo(now);
+
+    va = await this.vesting.vestedAmount(this.token.address);
+
+    va.should.bignumber.equal(amount.div(2));
+
```
Comment (xceltripofficial) [inline test/TestStepVestingPromo.js]: lets add this --
    let halfReleased = await this.token.balanceOf(beneficiary);
    halfReleased.should.bignumber.equal(amount.div(2));

File: test/TestStepVestingPromo.js [hunk 2]
```diff
@@ -37,25 +37,74 @@ contract('StepVesting', function ([_, owner, beneficiary]) {
...
+    now = this.start + this.cliffDuration + this.stepVestingDuration + 2;
+
+    await timeUtils.increaseTimeTo(now);
+
+    va = await this.vesting.vestedAmount(this.token.address);
+
+    va.should.bignumber.equal(amount);
+
+    await this.vesting.release(this.token.address).should.be.fulfilled;
```
Comment (xceltripofficial) [inline test/TestStepVestingPromo.js]: lets add this -

    let fullReleased = await this.token.balanceOf(beneficiary);
    fullReleased.should.bignumber.equal(amount);

Comment (ismaelmk) [reply test/TestStepVestingPromo.js]: done

File: test/TestStepVestingPromo.js (hunk 1 above)
Comment (ismaelmk) [reply test/TestStepVestingPromo.js]: Done

**Pull request merged**